package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "home",
-    "homepage": "https://<organization>.github.io/<repo>/",
+    "homepage": "https://womeninsoftwareengineeringjp.github.io",
```

Comment:
We can leave this as `<organization>` and it should do the right thing. removing `<repo>` makes sense now though